README.md

```diff
@@ -712,157 +716,87 @@ http://localhost:8116
     "objects": [
       {
         "block_number": "0x5b033a",
-        "cells": [
-          [
-            "output",
-            "0x0"
-          ]
-        ],
+        "cells": [["output", "0x0"]],
         "tx_hash": "0x556060b62d16386da53f8a4b458314dfa2d1988a7bcc5c96c3bb2a350a3453a1",
         "tx_index": "0x4"
       },
       {
         "block_number": "0x5b0671",
         "cells": [
-          [
-            "input",
-            "0x0"
-          ],
-          [
-            "output",
-            "0x1"
-          ]
+          ["input", "0x0"],
+          ["output", "0x1"]
```

Comment:
I think it would be better to retain the indentation and new line in there.

Reply:
Done